### PR TITLE
bump up the number of jobs Safari gets on Azure Pipelines to ensure completion

### PR DESCRIPTION
**Pull request by gsnedders**

### CODE REVIEW

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -479,7 +479,7 @@ jobs:
        eq(variables['Build.SourceBranch'], 'refs/heads/triggers/safari_stable'),
        and(eq(variables['Build.Reason'], 'Manual'), variables['run_all_safari']))
   strategy:
-    parallel: 5 # chosen to make runtime ~2h
+    parallel: 8 # chosen to make runtime ~2h
```
Comment (foolip) [inline .azure-pipelines.yml:482]: Is the comment still true after this bump?

Comment (gsnedders) [reply .azure-pipelines.yml:482]: Taking a recent complete run, maybe we should go for 7, but I was trying to leave a bit more of a margin this time?

**Review by nt1m — APPROVED** [review]
Seems reasonable given the number of jobs for Edge is 10. As @foolip mentions, is the runtime still approximately 2h?

File: .azure-pipelines.yml (hunk 1 above)
Comment (foolip) [reply .azure-pipelines.yml]: OK, sounds like you've looked at the total runtime at least. Thanks!

**Review by foolip — APPROVED** [review]

**Pull request merged**